app/uk/gov/hmrc/fhregistrationfrontend/actions/PageAction.scala

```diff
@@ -67,23 +66,23 @@ class PageAction[T, V] @Inject()(pageId: String, sectionId: Option[String], jour
     if (state.isPageComplete(page) || state.nextPageToComplete().contains(page.id)) {
       Right(true)
     } else {
-      Logger.error(s"Not found")
+      logger.error(s"Not found")
       Left(errorHandler.errorResultsPages(Results.NotFound))
     }
 
   def loadPageSection(page: Page[T])(implicit request: Request[_]): Either[Result, Page[T]] =
     if (page.withSubsection isDefinedAt sectionId)
       Right(page withSubsection sectionId)
     else {
-      Logger.error(s"Not found")
+      logger.error(s"Not found")
       Left(errorHandler.errorResultsPages(Results.NotFound))
     }
 
   def loadPage[A](request: JourneyRequest[A]): Either[Result, Page[T]] =
     request.journeyState.get[T](pageId) match {
       case Some(page) ⇒ Right(page)
       case None ⇒
-        Logger.error(s"Not found")
+        logger.error(s"Not found")
```

Comment:
not specific to this PR, but these generic `Not found` messages are no use :) , i would delete them, but its ok if you want to keep them

Reply:
mmm exactly am not sure why they are added here. we are really no idea. am happy to delete them.